tests/test_copilot_helper.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+from click.testing import CliRunner
+
+from copilot_helper import cli
+
+
+class TestCopilotHelperCli:
+    def test_check_version(self):
+        runner = CliRunner()
+
+        result = runner.invoke(cli, ["--version"])
```

Comment:
You could just inline the runner...
```suggestion
        result = CliRunner().invoke(cli, ["--version"])
```

Reply:
True dat. I thought I would be writing moar tests, but decided this was enough for now.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/51/commits/ddb762b327dfdc70273dd134078d043c96db5d46.